lib/katello/tasks/upgrades/3.0/update_subscription_facet_backend_data.rake

```diff
@@ -46,10 +46,8 @@ namespace :katello do
             subscription_facet.save!
 
             host = subscription_facet.host
-            if host.name.include?('_') || host.name != host.name.downcase || host.name.ends_with?('.')
-              host.name = host.name.gsub('_', '-').chomp('.').downcase
-              host.save!
-            end
+            host.name = ::Katello::Host::SubscriptionFacet.sanitized_name(host.name)
```

Comment:
No test, no failure! Granted it is an upgrade rake task, but this would fail s/sanitized_name/sanitize_name